src/store/actions/actions-model.ts

```diff
@@ -230,12 +226,8 @@ export const setProvisionsRemoved: ActionIF = ({ commit }, provisionsRemoved: bo
   commit('mutateProvisionsRemoved', provisionsRemoved)
 }
 
-export const setOriginalResolutionDates: ActionIF = ({ commit }, resolutionDates: string[]): void => {
-  commit('mutateOriginalResolutionDates', resolutionDates)
-}
-
-export const setResolutionDates: ActionIF = ({ commit }, resolutionDates: string[]): void => {
-  commit('mutateResolutionDates', resolutionDates)
+export const setNewResolutionDates: ActionIF = ({ commit }, resolutionDates: string[]): void => {
```

Comment:
The getter for this data is `getNewResolutionDates` (which kind of makes sense in how it's used), so I renamed this for symmetry.